util/topsql/state/state.go

```diff
@@ -48,6 +48,16 @@ type State struct {
 	ReportIntervalSeconds *atomic.Int64
 }
 
+// EnabledTopSQL enable the top SQL feature.
+func EnabledTopSQL() {
```

Comment:
This should be called `EnableTopSQL`. `Enable` is a verb. `Enabled` is a state.

Reply:
great